app/views/movies/_movie_partial_loop.html.erb

```diff
@@ -4,17 +4,21 @@
       <article class="tile-cover-pic">
         <!-- MODAL TRIGGER -->
 
-        <% if @list.present? %>
-          <p><%= link_to image_for(movie), movie_modal_path(tmdb_id: movie.tmdb_id, list_id: @list.id), remote: true, data: { target: "#myModal_#{movie.tmdb_id}" }, id: "modal_link_#{movie.tmdb_id}" %></p>
-        <% else %>
-          <p><%= link_to image_for(movie), movie_modal_path(tmdb_id: movie.tmdb_id), remote: true, data: { target: "#myModal_#{movie.tmdb_id}" }, id: "modal_link_#{movie.tmdb_id}" %></p>
-        <% end %>
+        <p id="modal-trigger-mobile">
+          <%= link_to image_for(movie), movie_path(movie) %>
+        </p>
 
-        <div id="movies_partial_<%= "#{movie.tmdb_id}" %>">
+        <div id="modal-trigger">
+          <% if @list.present? %>
+            <p><%= link_to image_for(movie), movie_modal_path(tmdb_id: movie.tmdb_id, list_id: @list.id), remote: true, data: { target: "#myModal_#{movie.tmdb_id}" }, id: "modal_link_#{movie.tmdb_id}" %></p>
+          <% else %>
+            <p><%= link_to image_for(movie), movie_modal_path(tmdb_id: movie.tmdb_id),                    remote: true, data: { target: "#myModal_#{movie.tmdb_id}" }, id: "modal_link_#{movie.tmdb_id}" %></p>
+          <% end %>
+        </div>
```

Comment:
It looks like this whole if/else statement is only determining if it should pass `list_id` as a param.

So I think we can simplify it by just using safe navigator on `@list&.id`, and remove the if/else (I tested it locally):

```html
<div id="modal-trigger">
   <p><%= link_to image_for(movie), movie_modal_path(tmdb_id: movie.tmdb_id, list_id: @list&.id), remote: true, data: { target: "#myModal_#{movie.tmdb_id}" }, id: "modal_link_#{movie.tmdb_id}" %></p>
</div>
```
I know you didn't edit that bit of code, but since the file is being edited, I was thinking we may as well clean it up. I think it could be done in this PR, but if you'd rather leave it for a separate issue, that's fine w/ me.

Reply:
I'm cool with adding this change since you tested it locally. I wasn't sure if the presence of the param key with a nil value would cause issues. But if we're cool, i'm game.